src/Avalonia.OpenGL/Egl/EglInterface.cs

```diff
@@ -63,7 +63,7 @@ public EglInterface(string library) : base(Load(library))
         
         public delegate IntPtr EglGetProcAddress(Utf8Buffer proc);
         [GlEntryPoint("eglGetProcAddress")]
-        public EglGetProcAddress GetProcAddress { get; }
+        new public EglGetProcAddress GetProcAddress { get; }
```

Comment:
Not sure if this is appropriate, but in general MSDN uses `public new T MethodName` instead of `new public T MethodName`, according to https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/knowing-when-to-use-override-and-new-keywords See also https://stackoverflow.com/a/33237262/11351183